packages/test-helpers/src/utils/numbers.js

```diff
@@ -1,6 +1,7 @@
 const { BN } = require('web3-utils')
 
 const bn = x => new BN(x)
+const pct = x => bigExp(x, 16)
```

Comment:
I've taken the liberty of renaming this to `pct16()` in #40, as it's based on 1% being 1e16.